server/src/main/kotlin/eu/kanade/tachiyomi/network/interceptor/CloudflareInterceptor.kt

```diff
@@ -56,11 +57,38 @@ class CloudflareInterceptor(
             originalResponse.close()
             // network.cookieStore.remove(originalRequest.url.toUri())
 
-            val request =
+            val flareResponseFallback = serverConfig.flareSolverrAsResponseFallback.value
+            val flareResponse =
                 runBlocking {
-                    CFClearance.resolveWithFlareSolverr(setUserAgent, originalRequest)
+                    CFClearance.resolveWithFlareSolver(originalRequest, !flareResponseFallback)
```

Comment:
Why was setUserAgent removed?

Reply:
It is not removed but simply moved to elsewhere. In the case of `flareResponseFallback` being set it's called when the flareSolver browser received a succesful response. Otherwise it's applied in `requestWithFlareSolverr` as normal.